### DISCUSSION TITLE
Some modes unavailable when roads are upgraded with tram tracks

### DISCUSSION THREAD

**Discussion by Matikson** [slyh/Cities2-TrafficLightsEnhancement#56] [Q&A] [answered]
I want to make a good tram network in my city, but vanilla traffic lights are just awful. So I installed the mod, but for some reason other modes than Vanilla or Split Phasing are just not there... On the same save, junctions that don't have any tram tracks coming in, work flawlessly (except my traffic handling skills). I'm just asking to know, if a workarround I found actually has a chance to work, and if is this intentional. The workarround is to change the mode before upgrading the roads with the tracks. 
![obraz](https://github.com/slyh/Cities2-TrafficLightsEnhancement/assets/127886507/0cbd4349-050c-44a1-bb97-33f214a3effe)
P.S. I tested this also with non-devided roads and they have the same issue.

**Comment by slyh** [comment, marked as answer]
It is intentional to reduce the number of modes available in complex junctions to avoid confusion. I recognise this may seem like a downgrade in functionality. I will explore ways to give players more control.

**Reply by Matikson** [reply]
Thank you, it would be rea6 appreciated!
